.tool-versions

```diff
@@ -1 +1 @@
-ruby 2.7.6
\ No newline at end of file
+ruby 3.1.2
```

Comment:
This tells [asdf](https://asdf-vm.com) which Ruby version to use and got missed in the last upgrade.